### DISCUSSION TITLE
<title>Prevent blob pruning with a temporary commit</title>

### DISCUSSION THREAD

**Discussion by X01XX** [community/community#88469] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When updating files, I get an email for each file, even if nothing changed, saying  "Prevent blob pruning with a temporary commit".  Since I have around 30 files, this is annoying.

**Comment by ProditorMagnus** [comment]
Solved with email filter `"Branch: refs/__gh__/temp/upload-manifest/"`

**Reply by X01XX** [reply]
Thanks, but still got:
 Changed paths:
    A regionstorecorr.rs

  Log Message:
  -----------
  Prevent blob pruning with a temporary commit

**Comment by bhmoto** [comment]
UI surface for this deprecation protocol now tracked as issue #1964 — Infrastructure tab with `current` / `deprecated` / `target` node state tagging. Deprecated nodes going dark IS the progress signal.
